solidity/test/DepositLiquidationTest.js

```diff
@@ -163,23 +163,25 @@ describe("DepositLiquidation", async function() {
       ).to.eq.BN(tokenCheck)
     })
 
-    it("distributes value to the buyer", async () => {
-      const value = 10000000000000000
+    it("awards withdrawable value to the buyer", async () => {
+      const value = new BN("10000000000000000")
       const block = await web3.eth.getBlock("latest")
       const notifiedTime = block.timestamp
-      const initialBalance = await web3.eth.getBalance(buyer)
 
       await ecdsaKeepStub.pushFundsFromKeep(testDeposit.address, {value: value})
 
       await testDeposit.setLiquidationAndCourtesyInitated(notifiedTime, 0)
+      const auctionValue = await testDeposit.auctionValue()
+
       await testDeposit.purchaseSignerBondsAtAuction({from: buyer})
 
-      const finalBalance = await web3.eth.getBalance(buyer)
+      const withdrawable = await testDeposit.getWithdrawAllowance.call({
+        from: buyer,
+      })
 
-      expect(
-        new BN(finalBalance),
-        "buyer balance should increase",
-      ).to.be.gte.BN(initialBalance)
+      expect(withdrawable, "buyer should have a withdrawable balance").to.eq.BN(
+        auctionValue,
+      )
```

Comment:
Ditto here on asserting the deposit has this amount in escrow.

Reply:
In this case, we need `contractBalance = purchesedBond + remainder / 2` since we're not in fraud.